src/dfx/src/commands/canister/install.rs

```diff
@@ -22,7 +24,7 @@ pub struct CanisterInstallOpts {
     canister: Option<String>,
 
     /// Deploys all canisters configured in the project dfx.json files.
-    #[clap(long, required_unless_present("canister"))]
+    #[clap(long, short, required_unless_present("canister"))]
```

Comment:
What is the rationale for adding `short` here?  We have very few command-line options that provide a short version, and even some of those are dubious.

Reply:
Oh, that was left over from testing. If you do not want to keep it I can just revert it.

Reply:
Yeah, let's revert it please